Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9495,10 +9495,13 @@ def get_current_time(time_zone=0):
 
 def filter_incidents_by_duplicates_and_limit(incidents_res, last_run, fetch_limit, id_field):
     """
-    Remove duplicate incidents from response and returns the incidents till limit
+    Removes duplicate incidents from response and returns the incidents till limit.
+    The function should be called after getting the get-incidents API response,
+    and by passing the id_field it will filter out the incidents that were already fetched
+    by checking the incident IDs that are saved from the previous fetch in the last run object
 
     :type incidents_res: ``list``
-    :param incidents_res: The incidents from response
+    :param incidents_res: The incidents from the API response
```

Comment:
```suggestion
    :param incidents_res: The incidents from the raw API response
```
couldn't this be also incidents that were parsed after the api response?
for example i'd have to parse the api response in case i have nested ```id_field``` in the raw response in order to send the incidents to the ```filter_incidents_by_duplicates_and_limit``` function

Reply:
It can be any list of objects that contains the id_field